stdlib/runtime/numbers.gr

```diff
@@ -2508,6 +2508,88 @@ provide let (>>) = (x: Number, y: Number) => {
 // [NOTE]: Coercion is a *conservative* process! For example, even if a float is 1.0,
 //         we will fail if attempting to coerce to an int!
 
+/**
+ * Converts a Number to an Int32.
+ *
+ * @param number: The value to convert
+ * @returns The Number represented as an Int32
+ *
+ * @since v0.2.0
```

Comment:
Wrong since?

Reply:
Yeah looks like I messed up the comment too :sweat_smile: